### DISCUSSION TITLE
Suggestion: List of Quality of Life Improvements

### DISCUSSION THREAD

**Discussion by KieranP** [SigNoz/signoz#6329] [General]
Having used Signoz in production environment for about 2 months now, I've compiled a list of items that would make a huge improvement to the daily usage of Signoz. This does not include entirely new functionality/pages, but changes/tweaks to existing functionality. I will continue to add to this list over time.

My hope is that the Signoz team might be able to work through a couple of these each release, greatly improving the overall experience. The following are broken up by type/section of the Signoz app.

**Site Wide**

- [x] Bug: Turn autocomplete off for the time selectors fields (currently clicking shows dropdown of 1Password accounts for the domain, which interferes with the UI)

**Services**

- [ ] Tweak: Allow sorting by the application column

**Traces**

- [x] Feature: Implement the updated query builder widget that Logs uses (with color coded filter attributes)
- [ ] Feature: Ability to star/bookmark important Views
- [x] Feature: Ability to customize which quick filters to show

**Logs**

- [ ] Bug: Missing the refresh button (like the one that is available on the Traces dashboard)
- [ ] Feature: Ability to star/bookmark important Views
- [x] Feature: Ability to customize which quick filters to show

**Dashboards**

- [ ] Tweak: Allow sorting by dashboard name
- [ ] Tweak: Ability to star/bookmark certain dashboards
- [x] Tweak: When adding a dashboard panel using "New Panel" button, add the new panel to the bottom of the dashboard rather than the top (edit: fixed in https://github.com/SigNoz/signoz/pull/6993)
- [x] Tweak: When a dashboard is using sections, allow adding a new panel to bottom of a chosen section (by clicking the three dots on the section bar and clicking "New Panel" from the dropdown) (edit: fixed in https://github.com/SigNoz/signoz/pull/6999)

**Comment by srikanthccv** [comment]
Created issue
